bootstrap/cmd/kfctl/cmd/init.go

```diff
@@ -15,136 +15,28 @@
 package cmd
 
 import (
-	"errors"
+	"fmt"
 
-	kftypes "github.com/kubeflow/kubeflow/bootstrap/v3/pkg/apis/apps"
-	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
+	"github.com/fatih/color"
 )
 
-var initCfg = viper.New()
-
+// DeprecatedPrint sets the color for deprecation warnings
+var DeprecatedPrint = color.New(color.FgYellow).SprintFunc()
+var initDeprecationMessage = DeprecatedPrint("'kfctl init' has been DEPRECATED.") + "\n" +
```

Comment:
Would it be better to say "removed" rather than DEPRECATED?

Reply:
done